java/tools/src/java/org/apache/orc/tools/RowCount.java

```diff
@@ -66,7 +66,9 @@ public static void main(Configuration conf, String[] args) throws Exception {
         }
       }
     }
-    System.exit(bad == 0 ? 0 : 1);
+    if (bad > 0) {
+      System.exit(1);
+    }
```

Comment:
Please revert this.

Reply:
Before Java17, we could use `SecurityManager` to intercept `System.exit` for testing, but we are in Java17 now, so I will propose another PR to avoid `System.exit(0)`.

https://www.dontpanicblog.co.uk/2022/10/30/testing-system-exit/

https://openjdk.org/jeps/411

Reply:
You cannot proceed a `main` code change in a test PR, `Add test for ...`.

Reply:
You're right, thanks for the comment. @dongjoon-hyun